src/page/address-step.page.ts

```diff
@@ -1,10 +1,10 @@
-import { $, ElementFinder, browser, ExpectedConditions } from 'protractor';
+import { ElementFinder, browser, ExpectedConditions, by, element } from 'protractor';
 
 export class AddressStepPage {
   private cartAddressNavigation: ElementFinder;
 
   constructor() {
-    this.cartAddressNavigation = $('#center_column > form > p > button > span');
+    this.cartAddressNavigation = element(by.name('processAddress'));
```

Comment:
try to use `element `or `$()` to get the elements but avoid to use both

Reply:
Ok. I just found $( )  is a shortcut to element( by.css(  ) )